terraform/datadog/concourse.tf

```diff
@@ -61,7 +61,7 @@ resource "datadog_timeboard" "concourse-jobs" {
     viz   = "timeseries"
 
     request {
-      q = "${format("avg:concourse.pipeline_time{bosh-deployment:%s,pipeline_name:create-cloudfoundry}", var.env)}"
+      q = "${format("avg:concourse.pipeline_time{deploy_env:%s,pipeline_name:create-cloudfoundry}", var.env)}"
```

Comment:
for this to work, https://github.com/alphagov/paas-cf/blob/master/concourse/pipelines/create-cloudfoundry.yml#L2198 needs to use deploy_env tag